### PR TITLE
Remove unused BaseSshUriProperties

### PR DESCRIPTION
**Pull request by shakuzen**
I believe this class was added by mistake in merge commit 7e901a1d4f951c829a6a27a0b3618218fbf3417e. It is unused and appears to be mostly a duplicate of `SshUri`.

### CODE REVIEW

File: docs/pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	<parent>
 		<groupId>org.springframework.cloud</groupId>
 		<artifactId>spring-cloud-config</artifactId>
-		<version>1.3.3.BUILD-SNAPSHOT</version>
+		<version>1.4.0.BUILD-SNAPSHOT</version>
```
Comment (shakuzen) [inline docs/pom.xml]: While this change is unrelated, the build would not pass for me without making this change since it referenced the wrong parent version.

Comment (ryanjbaxter) [inline docs/pom.xml]: I will fix this in a separate commit

**Comment by ryanjbaxter** [comment]
Hum I am unsure where that class is coming from @dsyer any ideas?

**Comment by dsyer** [comment]
I guess Tommy is probably right (left over from a messy merge).

**Comment by ryanjbaxter** [comment]
Thanks @dsyer.  @shakuzen can you update your branch with the latest from `master` and then I will merge this.  Thanks!

**Comment by shakuzen** [comment]
@ryanjbaxter Done
